pkgs/development/python-modules/jmp/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, jax
+, jaxlib
+}:
+
+buildPythonPackage rec {
+  pname = "jmp";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "deepmind";
+    repo = "jmp";
+    rev = "v${version}";
+    sha256 = "1kah1m3v08gmhk1q81iwpyb5q5rxcc0wln7w0kr5zvpml61k7gxp";
+  };
+
+  # Wheel requires only `numpy`, but the import needs both `jax` and `jaxlib`.
+  propagatedBuildInputs = [
+    jax
+    jaxlib
+  ];
+
+  pythonImportsCheck = [
+    "jmp"
+  ];
+
+  # checkInputs = [
+  #   pytest
+  # ];
+
+  # The tests are currently broken both for release and `master`: `AttributeError: module 'jmp' has no attribute '_src'`
+  doCheck = false;
```

Comment:
if this has native extensions, then you probably need

```
preCheck = ''
  export PYTHONPATH=$out/${python.sitePackages}:$PYTHONPATH
'';
```

To have the compiled extensions available

Reply:
Also we should link to an upstream issue here, or create one if it does not exist yet.

Reply:
Apparently there were two issues: 1) Running the tests without `pytestCheckHook`; 2) Stable `jmp` version breaks on recent JAX versions. Fixed both.

Reply:
ok, great!